webpack.production.js

```diff
@@ -83,7 +85,8 @@ module.exports = ( env ) => {
 							rimraf.sync( './dist/*.LICENSE.txt' );
 						} );
 					}
-				} )()
+				} )(),
+				new BannerSizeWarningPlugin( { limit: 25 * 1024 } ) // compiled banner size (in bytes)
```

Comment:
probably caused by testing, remember to set it back to 250

Reply:
Fixed it. Sorry, will be careful of such things next time.